spring-boot-modules/spring-boot-bootstrap/pom.xml

```diff
@@ -47,13 +48,12 @@
         <dependency>
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
-            <version>${rest-assured.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>${servlet.version}</version>
```

Comment:
Is that needed?
After removing it, module still builds fine

Reply:
That dependency was removed. This appear in the removed items.